manifests/windows/install.pp

```diff
@@ -12,8 +12,17 @@
     default   => $::puppet_agent::arch
   }
 
+  if $::puppet_agent::is_pe {
```

Comment:
So crazy question, once someone is on the AIO agent and uses this for the next upgrade...